src/renderer/components/+events/events.scss

```diff
@@ -1,4 +1,10 @@
 .Events {
+  h5.title {
+    .events-count {
+      font-size: $font-size;
+    }
+  }
+
```

Comment:
No need to change font size. It's better to move `events-count` out from `<h5>`
![events table](https://user-images.githubusercontent.com/9607060/107778849-a0104280-6d55-11eb-946a-499fc07cc067.png)

Reply:
Fixed / moved to `customizeHeader` prop.